landbosse/model/CollectionCost.py

```diff
@@ -512,29 +612,33 @@ def calc_total_cable_length(total_turbines, count, check_terminal,  turbines_per
             # If total turbines in project are less than cumulative turbines
             # up till and including that cable.
 
-            terminal_string = cable.turb_sequence - 1  # Flag this cable as it is
-            # also the actual terminal cable
+            if total_turbines <= turbines_per_cable[count]:
 
-            if (cable.turb_sequence - 1) == 0:  # That is, if cable # 1 can hold
-                # more turbines than specified by user, it is the terminal cable
+                terminal_string = cable.turb_sequence - 1  # Flag this cable as it is
```

Comment:
I don't think we need terminal cable anymore, because that was substation-interconnect, right?